libs/traits/src/rewards.rs

```diff
@@ -56,7 +62,7 @@ where
 	fn distribute_reward<It>(
 		reward: Self::Balance,
 		groups: It,
-	) -> Result<Vec<(Self::GroupId, DispatchError)>, DispatchError>
+	) -> Result<Vec<Result<Self::Balance, DispatchError>>, DispatchError>
```

Comment:
This new type is much more relevant and allows the tests to know exactly how much reward has been minted.